### PR TITLE
halloween theme

### PR DESCRIPTION
**Pull request by OzoneH3**
#### Summary
```SUMMARY: Interface "Halloween CDDA main menu logo"```

#### Purpose of change
Trick or Treat?!

#### Additional context
```
Toggleable in main_menu.cpp
static const bool halloween_theme = true;
```

### CODE REVIEW

File: src/main_menu.cpp [hunk 1]
```diff
@@ -99,26 +101,27 @@ void main_menu::print_menu( const catacurses::window &w_open, int iSel, const in
     const nc_color cColor2 = c_light_blue;
     const nc_color cColor3 = c_light_blue;
 
+    std::string marker = "";
```
Comment (BevapDin) [inline src/main_menu.cpp]: You don't need to initialize a string with an empty C-String. It's automatically created as empty string.

File: src/main_menu.cpp [hunk 2]
```diff
@@ -1116,3 +1128,48 @@ void main_menu::world_tab()
...
+        "  /_/ {<color_c_red>..</color>} \\_\\\n"
+        "  { {      } }\n"
+        "  , ,      , .";
+
+    return spider;
+}
+
+std::string main_menu::halloween_graves()
+{
+    static const std::string spider =
```
Comment (ZhilkinSerg) [inline src/main_menu.cpp]: spider instead of graves?

**Comment by Soyweiser** [comment]
http://prntscr.com/l8ag0v this is how it looked on the minimum screen size :D

**Comment by AMurkin** [comment]
RIP RIP

**Pull request merged**